lib/rspec/core/formatters/exception_presenter.rb

```diff
@@ -254,7 +271,7 @@ def multiple_exceptions_error?(exception)
           end
 
           def multiple_exception_summarizer(exception, prior_detail_formatter, color)
-            lambda do |example, colorizer, indentation|
+            lambda do |example, colorizer|
```

Comment:
Nice to see things getting simpler here with fewer variables needed in these lambdas...